python/src/ConditionalDistribution_doc.i.in

```diff
@@ -85,7 +78,7 @@ Draw a sample:
 // ---------------------------------------------------------------------
 
 %feature("docstring") OT::ConditionalDistribution::getConditionedDistribution
```

Comment:
The formatting is weird: why are the returns bold?

![image](https://github.com/openturns/openturns/assets/31989332/fe438034-b136-40f9-abcb-d0b97f8bb30a)